common/dynamicconfig/collection_test.go

```diff
@@ -134,32 +123,23 @@ func (s *collectionSuite) TestGetIntPropertyFilteredByTaskQueueInfo() {
 }
 
 func (s *collectionSuite) TestGetFloat64Property() {
-	setting := GlobalFloatSetting{
-		key: testGetFloat64PropertyKey,
-		def: 0.1,
-	}
+	setting := NewGlobalFloatSetting(testGetFloat64PropertyKey, 0.1, "")
```

Comment:
Nit: can we separate the tests from the main code module?

Reply:
What do you mean? Like, use a `_test` package?

Reply:
Yeah

Reply:
I'll follow up in another PR